utils/frame/frame-utilities-cli/Cargo.toml

```diff
@@ -18,6 +18,7 @@ sc-cli = { version = "0.10.0-dev", path = "../../../client/cli" }
 sp-runtime = { version = "6.0.0", path = "../../../primitives/runtime" }
 frame-system = { version = "4.0.0-dev", path = "../../../frame/system" }
 frame-support = { version = "4.0.0-dev", path = "../../../frame/support" }
+lazy_static = "1.4.0"
```

Comment:
```suggestion
```

Reply:
How did this end up here :see_no_evil:, thanks for the catch.